src/@types/FileInfo.ts

```diff
@@ -36,6 +36,12 @@ export interface FileInfo {
    */
   method?: string
 
+  /**
+   * Computed file checksum
+   * @type {string}
+   */
+  checksum?: string
```

Comment:
I don't understand why everything in this typing is optional, and now also the `checksum`. Based on the typings this is the only response I should expect:

```js
const { type } = getFileInfo()
```

Because everything else is optional. What exactly am I supposed to do with the `type` alone? Why would all the other fields be empty int he response?

Reply:
yeah they kind of are optionals based on various responses we receive for provider when we query this. As i understood we will break this into smaller types and work with inheritance once we are going to integrate other storage options 😃